lib/screens_web/controllers/screen_api_controller.ex

```diff
@@ -32,6 +32,8 @@ defmodule ScreensWeb.ScreenApiController do
     _ = Screens.LogScreenData.log_data_request(screen_id, last_refresh, is_screen)
 
     if nonexistent_screen?(screen_id) do
+      Screens.LogScreenData.log_api_response(:nonexistent, screen_id, last_refresh, is_screen)
```

Comment:
I see that one of the done criteria in the ticket is to only log for "real" screens. I'm making the assumptions that nonexistent == not real and that in this case, `is_screen` would be `false`. If those assumptions are true, would we be able to remove this call to `Screens.LogScreenData.log_api_response/4` here (and in v2) and also remove the `if is_screen do` check on line 54 of `lib/screens/log_screen_data.ex`?

Reply:
So `is_screen` is a request parameter that is passed to the server. That parameter is set for live screens in whatever CMS it is using. So this specific response could still happen for live screens. Will be uncommon but happens mostly when we take screens out of the config temporarily for big config changes (and even that happens mostly in dev and dev-green). 

And since it is a parameter that we won't ever set when working locally (unless we need to test logging), then we want to keep the conditional so we don't log while pulling up screens in the browser.

Reply:
Gotcha, thanks for the explainer!

So does that imply that we would want to be logging responses for nonexistent screens while they're temporarily out of the config? And if that's correct, what's the reasoning behind that?

Reply:
I think the main goal of the response logging is to compare the number of responses to the number of requests. So if there is a request for the nonexistent screen, we will want this log just to show the server did respond.